spec/services/medication_dispensation_service_spec.rb

```diff
@@ -12,7 +12,45 @@
 
```

Comment:
Let's explicitly make this patient hypertensive

Reply:
ok